core/src/main/res/layout/player_widget.xml

```diff
@@ -22,7 +23,7 @@
             android:background="?android:attr/selectableItemBackground"
             android:scaleType="fitCenter"
             android:padding="8dp"
-            android:src="@drawable/ic_av_play_white_24dp" />
+            app:srcCompat="@drawable/ic_av_play_white_24dp" />
```

Comment:
Are you sure that this works on all Android versions? Widgets use RemoteViews and I am not sure if that is compatible with srcCompat.

Reply:
See above.